src/notaryUtils.js

```diff
@@ -26,11 +26,15 @@ function createCaseID () {
  */
 function setupCase (params) {
   return co(function * () {
-    const uniqueID = params.caseID || uuid()
+    // the caseID param will be in the format of notary/cases/uuid,
+    // only check if its uuid is less than 40 characters
+    const uniqueID = params.caseID ? params.caseID.substring(params.caseID.indexOf('/cases/') + 7) : uuid()
```

Comment:
We are only "throwing" it away for the uniqueID check. If you see on line 37, we are using the URI in its entirety if provided